pkg/backend/oss.go

```diff
@@ -141,7 +142,9 @@ func (b *OSSBackend) push(ra content.ReaderAt, blobDigest digest.Digest) error {
 		}
 
 		if err := g.Wait(); err != nil {
-			b.bucket.AbortMultipartUpload(imur)
+			if err := b.bucket.AbortMultipartUpload(imur); err != nil {
+				log.L.WithError(err)
```

Comment:
Does this print message to stderr? We'd better not do this in a public package.

Reply:
No. It prints to logger

Reply:
fixed